src/icons/general/IconClockPreview.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+import type { Icon } from '../flowTypes';
+
+const IconClockPreview = ({ className = '', color = '#979797', height = 16, title, width = 16 }: Icon) => (
+    <AccessibleSVG
+        className={`icon-clock-preview ${className}`}
```

Comment:
Minor: Do we have the classNames util as to not have an empty space in the class if the default is used?

Reply:
updated